packages/owl-bot/src/owl-bot.ts

```diff
@@ -64,6 +64,19 @@ async function acquireLock(target: string): Promise<DatastoreLock> {
   return lock;
 }
 
+async function releaseLock(lock: DatastoreLock): Promise<void> {
+  try {
+    await lock.release();
+  } catch (_err: unknown) {
+    const err = _err as Error;
+    if (err.message.includes('DatastoreLockError')) {
```

Comment:
nit: would checking the error `instanceof` be more resilient than relying on the message string?

Reply:
Yes, but this would require exposing the Error from `DatastoreLock` and refactoring `DatastoreLock` to use a typed error.

I wasn't sure if it's worth the effort.